v19.2/tutorials/multi-region-setup.md

```diff
@@ -0,0 +1,157 @@
+    ~~~ shell
+    $ cockroach demo \
+    --nodes=9 \
+    --demo-locality=region=gcp-us-east1:region=gcp-us-east1:region=gcp-us-east1:region=gcp-us-west1:region=gcp-us-west1:region=gcp-us-west1:region=gcp-europe-west1:region=gcp-europe-west1:region=gcp-europe-west1
+    ~~~
+
+    ~~~
+    root@127.0.0.1:<some_port>/movr>
+    ~~~
+
+    Keep this terminal window open. Closing it will shut down the virtual cluster.
+
+1. Copy the connection string at the prompt (e.g., `root@127.0.0.1:<some_port>/movr`).
+
+1. In a separate terminal window, run the following command to load `dbinit.sql` to the demo database:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ cockroach sql --insecure --url='postgresql://root@127.0.0.1:<some_port>/movr' < dbinit.sql
+    ~~~
+
+    This file contains the `movr` database definition, and SQL instructions to geo-partition the database.
+
+1. In the open SQL shell to the virtual cluster, verify that the database schema loaded properly:
+
+    {% include copy-clipboard.html %}
+    ~~~ sql
+    > SHOW TABLES;
+    ~~~
+    ~~~
+      table_name
+    +------------+
+      rides
+      users
+      vehicles
+    (3 rows)
+    ~~~
+
+
+## Setting up a virtual development environment
+
+In production, you want to [containerize](https://www.docker.com/resources/what-container) your application and deploy it with a deployment orchestration tool, like [Kubernetes](https://kubernetes.io/). For debugging, use [`pipenv`](https://docs.pipenv.org/en/latest/install/#installing-pipenv), a tool that manages dependencies with `pip` and creates virtual environments with `virtualenv`.
+
+1. Run the following command to initialize the project's virtual environment:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ pipenv --three
+    ~~~
+
+    `pipenv` creates a `Pipfile` in the current directory. Open this `Pipfile`, and confirm its contents match the following:
+
+
+    ~~~ toml
+    [[source]]
+    name = "pypi"
+    url = "https://pypi.org/simple"
+    verify_ssl = true
+
+    [dev-packages]
+
+    [packages]
+    cockroachdb = "*"
+    psycopg2-binary = "*"
+    SQLAlchemy = "*"
+    SQLAlchemy-Utils = "*"
+    Flask = "*"
+    Flask-SQLAlchemy = "*"
+    Flask-WTF = "*"
+    Flask-Bootstrap = "*"
+    Flask-Login = "*"
+    WTForms = "*"
+    gunicorn = "*"
+    geopy = "*"
+
+    [requires]
+    python_version = "3.7"
+    ~~~
+
+1. Run the following command to install the packages listed in the `Pipfile`:
+
+    {% include copy-clipboard.html %}
+    ~~~ shell
+    $ pipenv install
+    ~~~
+
+1. To connect to a SQL database (like CockroachDB) from a client, you need a [SQL connection string](https://en.wikipedia.org/wiki/Connection_string). Rather than hard-coding the connection string into the source code, the application reads it from an environment variable. Pipenv automatically sets any variables defined in a `.env` file as environment variables in a Pipenv virtual environment.
```

Comment:
Recommend linking "connection string" to our [Connection parameters reference page](https://www.cockroachlabs.com/docs/stable/connection-parameters.html)